src/vm/amd64/cgenamd64.cpp

```diff
@@ -193,6 +193,15 @@ void HelperMethodFrame::UpdateRegDisplay(const PREGDISPLAY pRD)
         ENUM_CALLEE_SAVED_REGISTERS();
 #undef CALLEE_SAVED_REGISTER
 
+// In this case InsureInit called LazyMachState::unwindLazyState to fill in data.
+// But in the case of amd64 we don't have context pointers (see LazyMachState::unwindLazyState at gmsamd64.cpp:123)
+// So we'll just set it to null.
+#define CALLEE_SAVED_REGISTER(regname) pRD->pCurrentContextPointers->regname = NULL;
```

Comment:
It seems it would be a bit better to assign pUnwoundState->m_Ptrs.regname instead of NULL here and put the NULL assignment stuff to the LazyMachState::unwindLazyState in gmsamd64.cpp at line 126 as follows. That way, we fix it at the source and if we ever find a way to get context pointers for the debugger too, it would be a more obvious place to change.
```C++
#define CALLEE_SAVED_REGISTER(regname) unwoundState->m_Ptrs.p##regname = NULL;
    ENUM_CALLEE_SAVED_REGISTERS();
#undef CALLEE_SAVED_REGISTER
```